repo/meta/schema/vX-repo-definitions.json

```diff
@@ -577,6 +585,14 @@
         "$ref": "#/dcosReleaseVersion",
         "description": "The minimum DC/OS Release Version the package can run on."
       },
+      "knownIssues": {
```

Comment:
I don't think `boolean` would fit the need here. If there are known issues, do we use something else to point users to what those issues are? Can we use string arrays?

Reply:
Sure. And what do we put in these string arrays ?

Reply:
sorry about that @takirala, the original field we wanted to work off was `hasKnownIssues` which can totally be a boolean and is much less ambiguous. Can we update to use that field? See earlier tickets:
- https://jira.mesosphere.com/browse/DCOS-58017
- https://jira.mesosphere.com/browse/DCOS-58030

Btw, all we're trying to signify to users is this package has issues with deploying cleanly with default settings. We don't need to list what those users are. The maintainers of the package should make sure that the package can be
1) deployed successfully with default configs or
2) if additional params are needed, they need to be clear about them in the description and make them required params